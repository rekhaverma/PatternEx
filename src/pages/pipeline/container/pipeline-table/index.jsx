export { default } from './pipeline-table.container';
