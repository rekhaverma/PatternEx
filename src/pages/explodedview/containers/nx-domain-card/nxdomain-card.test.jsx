import React from 'react';

import { NXDomainsCard } from './nxdomain-card.container';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<NXDomainsCard />', () => {
  describe('render: ', () => {
    it('should match with snapshot', () => {
      const props = {
        isNxDomainsDataLoaded: true,
        nxDomainsData: [],
      };
      const wrapper = shallow(<NXDomainsCard {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
