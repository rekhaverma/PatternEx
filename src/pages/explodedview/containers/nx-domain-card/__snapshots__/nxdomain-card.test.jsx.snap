// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NXDomainsCard /> render:  should match with snapshot 1`] = `
<Section
  className="nxDomain"
  dropDownOptions={Array []}
  dropDownState={false}
  dropdown={false}
  expandable={true}
  expandableView={
    <List
      behaviorType=""
      columnFormat={Array []}
      customClass={
        Object {
          "base": "details-card",
          "modifiers": Array [
            "+rows-highlight",
          ],
        }
      }
      data={Array []}
      endDate={"2018-01-01T23:59:59.999Z"}
      handlers={Object {}}
      pipeline=""
      startDate={"2018-01-01T00:00:00.000Z"}
      tabData={Array []}
      title="Domain"
      type="@@listComponent/nxdomains"
    />
  }
  filterFeatureList={[Function]}
  isDraggable={true}
  loaded={true}
  modalStyle={Object {}}
  onDropDownClick={[Function]}
  onDropDownSelection={[Function]}
  onModalOpen={[Function]}
  selectedDropDown={Array []}
  size="small"
  title="NX Domain"
>
  <NoData
    className="nodata"
    intlDefault="There is no data to display"
    intlId="global.nodata"
    intlValues={Object {}}
    message=""
    withIcon={true}
  />
</Section>
`;
